test/xref2/references_to_pages.t/run.t

```diff
@@ -21,5 +25,5 @@ Every references in `Bad_references` should not:
 Every references in `P` should resolve:
 
   $ odoc_print page-p.odocl | jq_scan_references
-  {"`Root":["M","`TUnknown"]}
-  {"`Dot":[{"`Root":["M","`TUnknown"]},"t"]}
+  {"`Resolved":{"`Identifier":{"`Root":[{"`RootPage":"test"},"M"]}}}
+  {"`Resolved":{"`Type":[{"`Identifier":{"`Root":[{"`RootPage":"test"},"M"]}},"t"]}}
```

Comment:
This changed because I've added back the missing `M` module. I think that's how the test was intended to be.